docs/source/development-guidelines.rst

```diff
@@ -36,12 +36,19 @@ The following section assumes you already have virtualenvwrapper and pip install
    $ pip install -r ./etc/requirements_dev.txt
    $ pip install -r ./etc/requirements_blaze.txt 
 
-Finally, you can build the C extensions by running:
+You can build the C extensions by running:
 
 .. code-block:: bash
 
    $ python setup.py build_ext --inplace
 
+In case you want to install the package locally and use ``zipline`` wrapper script:
+
+.. code-block:: bash
+
+   $ pip install .
```

Comment:
What do you think about recommending [`pip install -e .`](https://pip.pypa.io/en/stable/reference/pip_install/#editable-installs) here instead? Using an editable install means that if you make changes in the source tree, those changes will have an effect when you run zipline. Since this doc is in part targeting users that want to contribute to Zipline, I could imagine someone being confused if they do `pip install .`, make changes, and then don't see those changes have any effect.

Reply:
Agree. Just updated the commit.